.circleci/config.yml

```
@@ -508,220 +121,9 @@ jobs:
-      # Android Automatic Experimental
-      - android-automatic-build-experimental:
-          filters:
-            branches:
-              only:
-                - develop
-          requires:
-            - lint-testunit
-      - android-google-play-production-experimental:
-          requires:
-            - android-automatic-build-experimental
+            - e2e-build-android
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)